src/Standards/Squiz/Docs/WhiteSpace/FunctionClosingBraceSpaceStandard.xml

```diff
@@ -0,0 +1,73 @@
+<documentation title="Function Closing Brace Space">
+    <standard>
+    <![CDATA[
+    There should be a single blank line before the closing function brace.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: A single blank line before the closing function brace.">
+        <![CDATA[
+function outer() <em>{
+
+}</em>;
```

Comment:
```suggestion
}</em>
```

Stray semicolon. This is a named function, not a closure, so no need for the semicolon.